### PR TITLE
Feature/minimal  footer

### PR DESCRIPTION
**Pull request by maccesar21**
## Global Navigation

Adds ability to author a minimal footer, and styles it accordingly.

### CODE REVIEW

File: src/footer/js/footer.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default (data) => {
 
 	const $footer = $('footer',
 		{
-			class: `${prefix} ${data.hideMenus ? 'skinny-footer' : ''}`
+			class: `${prefix} ${data.hideMenus && 'skinny-footer'}`
```
Comment (paulcpederson) [inline src/footer/js/footer.js]: if hide menus is false this will output the word false or undefined

File: src/footer/js/footer.js [hunk 2]
```diff
@@ -65,6 +65,8 @@ export default (data) => {
 		});
 
 		if (data.hideMenus) {
+			console.log(data.hideMenus);
```
Comment (paulcpederson) [inline src/footer/js/footer.js]: console log

File: src/footer/js/footer.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ export default (data) => {
 
 	const $footer = $('footer',
 		{
-			class: `${prefix} ${data.hideMenus && 'skinny-footer'}`
+			class: `${prefix} ${data.hideMenus ? 'skinny-footer' : 'false'}`
```
Comment (paulcpederson) [inline src/footer/js/footer.js]: I would use `''` instead of `false` this is going to put the class `false` in the dom, right?

Comment (maccesar21) [reply src/footer/js/footer.js]: @paulcpederson please review again..

Comment (paulcpederson) [reply src/footer/js/footer.js]: This will still put `false` in the dom, you should use an empty string `''` as I mentioned before...

**Comment by maccesar21** [comment]
@paulcpederson changes should be in place..

File: src/footer/js/footer.js [hunk 4]
```diff
@@ -34,11 +34,11 @@ export default (data) => {
 			$footerBreadcrumb,
 		),
 		$('div', {class: `${prefix}--wrapper`},
-			$('div', {class: `${prefix}-section--1 ${data.hideMenus ? 'hidden' : ''}`},
+			$('div', {class: `${prefix}-section--1 ${data.hideMenus && 'hidden'}`},
 				$footerBrand,
```
Comment (dlewis-irse) [inline src/footer/js/footer.js]: please revert this to -- $('div', {class: `${prefix}-section--1 ${data.hideMenus ? 'hidden' : ''}`},

File: src/footer/js/footer.js [hunk 5]
```diff
@@ -34,11 +34,11 @@ export default (data) => {
...
 				$footerSocial
 			),
-			$('div', {class: `${prefix}-section--2 ${data.hideMenus ? 'hidden' : ''}`},
+			$('div', {class: `${prefix}-section--2 ${data.hideMenus && 'hidden'}`},
 				$footerMenu
```
Comment (dlewis-irse) [inline src/footer/js/footer.js]: please revert this to -- $('div', {class: `${prefix}-section--2 ${data.hideMenus ? 'hidden' : ''}`},

**Pull request merged**